run-tests.sh

```diff
@@ -1,17 +1,45 @@
-#!/bin/sh
+#!/bin/bash
 #
 # This file is part of REANA.
 # Copyright (C) 2017, 2018 CERN.
 #
 # REANA is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
-pydocstyle reana_job_controller && \
-isort -rc -c -df **/*.py && \
-check-manifest --ignore ".travis-*" && \
-FLASK_APP=reana_job_controller/app.py flask openapi create openapi.json  && \
-diff -q openapi.json docs/openapi.json && \
-sphinx-build -qnN docs docs/_build/html && \
-python setup.py test && \
-rm openapi.json && \
-docker build -t reanahub/reana-job-controller .
+COMPONENT_NAME=reana-job-controller
+DOCKER_IMAGE_NAME=reanahub/$COMPONENT_NAME
+PLATFORM="$(python -c 'import platform; print(platform.system())')"
```

Comment:
I wonder whether we should have something similar in setup.py to avoid trying to install `htcondor` on macOS. For instance, if I want to build the docs a need to comment out `htcondor` line.

Reply:
https://github.com/reanahub/reana-job-controller/issues/210